### DISCUSSION TITLE
ReactiveUI validation for ComboBox not showing

### DISCUSSION THREAD

**Discussion by blemidon** [AvaloniaUI/Avalonia#14713] [General]
When I bind my combobox with SelectedValue instead of SelectedItem, the validation error text not showing below the ComboBox.
The validation itself working, just the text missing.
The same ComboBox with SelectedItem binding working perfectly.

Thanks in advance,
Imre

**Comment by maxkatz6** [comment]
I don't think ComboBox has any validation built-in, as it's a simple single-choice control, where there is nothing to validate on control level.
